plexapi/utils.py

```diff
@@ -323,7 +323,7 @@ def toDatetime(value, format=None):
                 return None
         else:
             try:
-                return datetime.utcfromtimestamp(0) + timedelta(seconds=int(value))
+                return datetime(1970, 1, 1) + timedelta(seconds=int(value))
```

Comment:
```suggestion
                return datetime.fromtimestamp(0) + timedelta(seconds=int(value))
```

See https://github.com/pkkid/python-plexapi/pull/1203#issuecomment-1745062063